pkg/synopsysctl/cmd_update.go

```diff
@@ -286,7 +286,11 @@ func updateOperator(namespace string, cmd *cobra.Command) error {
 		}
 	} else {
 		// create custom resource definitions
-		err = createCrds(namespace, isClusterScoped, newCrds)
+		crdConfigs, err := getCrdConfigs(operatorNamespace, isClusterScoped, crds)
+		if err != nil {
+			return err
+		}
```

Comment:
Need to verify the length of crdConfigs here.  Always possible someone tried to update w/o nothing deployed.  Should probably error in that case too, right?